test/scale-infra/INSTRUCTIONS.md

```diff
@@ -0,0 +1,131 @@
+### Examples of Individual Steps
+As an alternative to the quick-start deployment bootstrapping instructions, you can also run the individual steps starting from a local directory containing the git repo as follows:
+
+
+## Deployment
+1. Create the KubeStellar core infra on AWS:
+
+    First, set the following variables:
+
+    ```console
+    REGION="us-east-2"
+    VPC="<vpc_name>"
+    CLUSTER_NAME="core"
+    EC2_SSH_PUBLIC_KEY="<mykey>"
+    NUM_WORKER_NODES=2
+    EC2_INSTANCE_TYPE="t2.xlarge"
+    ARCH="x86_64"
+    EC2_AMI="<aws-ami>"
```

Comment:
It would be more convenient if all the lines that can be copy-and-pasted without editing are consecutive.

Reply:
Fixed!